packages/enterprise-search/test/integration/enterprise/index.test.ts

```diff
@@ -17,13 +17,25 @@
  * under the License.
  */
 
+import proxy from 'proxy';
+import * as http from 'http';
 import { test } from 'tap'
 import { Client } from '../../..'
+import { AddressInfo } from 'net';
 
 const url = process.env.ENTERPRISE_SEARCH_URL ?? 'http://localhost:3002'
 const username = process.env.ELASTIC_ENTERPRISE_USER ?? 'elastic'
 const password = process.env.ELASTIC_ENTERPRISE_PASSWORD ?? 'changeme'
 
+const createProxy = (): Promise<http.Server> => {
+  return new Promise((resolve, reject) => {
+    const server = proxy(http.createServer())
+    server.listen(0, '127.0.0.1', () => {
+      resolve(server)
+    })
+  })
+}
+
```

Comment:
We don't need this test since thery already tested in [elastic/elastic-transport-js](https://github.com/elastic/elastic-transport-js).

Reply:
Removed in 4f4c35a